styles/theme/colors/common.ts

```diff
@@ -2,7 +2,7 @@
 
 const common = {
 	primaryWhite: '#ECE8E3',
-	primaryGold: '#C9975B',
+	primaryGold: '#E4B378',
```

Comment:
`#C9975B` is the color I'm seeing in Figma for `Primary Gold`. Did this change?

Reply:
i tweaked it slightly for the dark theme.

Reply:
actually gonna revert this but keep the theme specific change